doozerlib/cli/release_gen_payload.py

```diff
@@ -129,12 +134,25 @@ def load_latest_builds(self):
 
         self.runtime.logger.info("Fetching latest image builds from Brew...")
         payload_images, invalid_name_items = self._get_payload_images(images)
-        self.state['payload_images'] = len(payload_images)
-        latest_builds, images_missing_builds = self._get_latest_builds(payload_images)
+        release_payload_images, non_release_items = self._get_non_release_images(payload_images)
+        self.state['payload_images'] = len(release_payload_images)
+        latest_builds, images_missing_builds = self._get_latest_builds(release_payload_images)
         self._designate_privacy(latest_builds)
         mismatched_siblings = self._find_mismatched_siblings(latest_builds)
 
-        return latest_builds, invalid_name_items, images_missing_builds, mismatched_siblings
+        return latest_builds, invalid_name_items, images_missing_builds, mismatched_siblings, non_release_items
+
+    def _get_non_release_images(self, images):
```

Comment:
The function name doesn't seem to reflect what this function is doing. Could you give it a better name?